sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CosmosDB Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CosmosDB Resources.")]
 
-[assembly: AssemblyVersion("1.3.3.0")]
-[assembly: AssemblyFileVersion("1.3.3.0")]
+[assembly: AssemblyVersion("1.3.5.0")]
+[assembly: AssemblyFileVersion("1.3.5.0")]
```

Comment:
This version should be the same as the version in CosmosDB.scproj.  (1.4.1.0 here )